demisto_client/demisto_api/api/default_api.py

```diff
@@ -4514,6 +4514,105 @@ def import_playbook_with_http_info(self, file, **kwargs):  # noqa: E501
             _request_timeout=params.get('_request_timeout'),
             collection_formats=collection_formats)
 
+    def import_reputation_handler(self, file, **kwargs):  # noqa: E501
+        """Import a reputation type  # noqa: E501
+
+        Import a reputation type to XSOAR  # noqa: E501
+        This method makes a synchronous HTTP request by default. To make an
+        asynchronous HTTP request, please pass async_req=True
+        >>> thread = api.import_reputation_handler(file, async_req=True)
+        >>> result = thread.get()
+
+        :param async_req bool
+        :param file file: file (required)
+        :return: ReputationsWithErrors
+                 If the method is called asynchronously,
+                 returns the request thread.
+        """
+        kwargs['_return_http_data_only'] = True
+        if kwargs.get('async_req'):
+            return self.import_reputation_handler_with_http_info(file, **kwargs)  # noqa: E501
+        else:
+            (data) = self.import_reputation_handler_with_http_info(file, **kwargs)  # noqa: E501
+            return data
+
+    def import_reputation_handler_with_http_info(self, file, **kwargs):  # noqa: E501
+        """Import a reputation type  # noqa: E501
+
+        Import a reputation type to XSOAR  # noqa: E501
+        This method makes a synchronous HTTP request by default. To make an
+        asynchronous HTTP request, please pass async_req=True
+        >>> thread = api.import_reputation_handler_with_http_info(file, async_req=True)
+        >>> result = thread.get()
+
+        :param async_req bool
+        :param file file: file (required)
+        :return: ReputationsWithErrors
+                 If the method is called asynchronously,
+                 returns the request thread.
+        """
+
+        all_params = ['file']  # noqa: E501
+        all_params.append('async_req')
+        all_params.append('_return_http_data_only')
+        all_params.append('_preload_content')
+        all_params.append('_request_timeout')
+
+        params = locals()
+        for key, val in six.iteritems(params['kwargs']):
+            if key not in all_params:
+                raise TypeError(
+                    "Got an unexpected keyword argument '%s'"
+                    " to method import_reputation_handler" % key
+                )
+            params[key] = val
+        del params['kwargs']
+        # verify the required parameter 'file' is set
+        if ('file' not in params or
+                params['file'] is None):
+            raise ValueError("Missing the required parameter `file` when calling `import_reputation_handler`")  # noqa: E501
+
+        collection_formats = {}
+
+        path_params = {}
+
+        query_params = []
+
+        header_params = {}
+
+        form_params = []
+        local_var_files = {}
+        if 'file' in params:
+            local_var_files['file'] = params['file']  # noqa: E501
```

Comment:
don't we raise ` ValueError` if it is not there?

Reply:
I think in the future I will see if we can bump the version of the tool we are using, but that will be a relatively big effort since everything that uses demisto-py will need to be validated.

Ideally a version bump will lead to better syntax here.